reflex/.templates/jinja/web/utils/context.js.jinja2

```diff
@@ -23,8 +23,11 @@ export const clientStorage = {}
 {% endif %}
 
 {% if state_name %}
+export const onLoadInternalEvent = () => [Event('{{state_name}}.{{const.on_load_internal}}')]
```

Comment:
also need `export const onLoadInternalEvent = () => []` in the `else` clause for stateless apps